guides/release_setup.md

```diff
@@ -62,11 +62,11 @@ Versioned releases can be one of two kinds:
    ```
 
 1. Run `./hack/generate-configs.sh` to regenerate
-   [config.yaml](../config/prow/jobs/config.yaml), otherwise the presubmit test
+   [config.yaml](../config/prod/prow/jobs/config.yaml), otherwise the presubmit test
    will fail. Merge such pull request and ask the
```

Comment:
<!--mdfmt-bot-->
Format markdown:
```suggestion
   [config.yaml](../config/prod/prow/jobs/config.yaml), otherwise the presubmit
   test will fail. Merge such pull request and ask the
```